src/components/molecules/Header.tsx

```diff
@@ -10,9 +10,10 @@ import { useStore } from '../../store/storeConfig';
 import RootStore from '../../store/root.store';
 import UserProfileHeader from './UserProfileHeader';
 import { makeStyles } from '@material-ui/core/styles';
+import LanguageButtons from '../molecules/LanguageButtons';
 
 const useStyles = makeStyles({
-  profileBox: {
+  settingsBox: {
```

Comment:
no need for the word "box". this name is used for styling only, and should not indicate the type of the element.
Like there is not need for `someClassNameSpan`  or  `someClassNameDiv`